### PR TITLE
Czech proofreading

### PR DESCRIPTION
**Pull request by baradrb**

### CODE REVIEW

File: cs/css/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
-Nechceme ale zase začínat úplně od začátku, že? Opět použijeme něco, co již bylo vytvořeno jinými programátory a sdíleno na internetu pro použití zdarma. Vymýšlet znovu, co již před námi někdo vymyslel, to by nebyla žádná legrace.
+Nechceme ale zase začínat úplně od začátku, že? Opět použijeme něco, co již bylo vytvořeno jinými programátory a sdíleno zdarma na internetu. Vymýšlet znovu, co již před námi někdo vymyslel, to by nebyla žádná legrace.
 
 ## Použijme Bootstrap!
 
 Bootstrap je jedním z nejpopulárnějších HTML a CSS frameworků pro vývoj pěkné webové stránky: https://getbootstrap.com/
 
-Byl napsán programátory, kteří pracovali pro Twitter a je nyní vyvíjeno dobrovolníky z celého světa.
+Napsali jo programátoři, kteří pracovali pro Twitter, a nyní ho vyvíjí dobrovolníci z celého světa.
```
Comment (tricoder42) [inline cs/css/README.md]: Typo

Comment (patjouk) [inline cs/css/README.md]: Can you tell me what the typo is? I'll change it and merge this PR :wink:

Comment (tricoder42) [inline cs/css/README.md]: > Napsali je programátoři, kteří pracovali pro Twitter, a nyní ho vyvíjí dobrovolníci z celého světa.

Thanks :) 👍

Comment (patjouk) [inline cs/css/README.md]: @baradrb can you add those changes to your PR? I thought I would be able to do it but this PR was made before the news github changes.

**Comment by olasitarska** [comment]
I understand this has been reviewed by @tricoder42, so I went ahead and merged it. Thanks!

**Pull request merged**